tests/ui/institutions/test_jsonresolvers.py

```diff
@@ -17,24 +17,12 @@
 
 """Test institutions jsonresolvers."""
 
-import pytest
-from flask import url_for
 
-from sonar.modules.documents.api import DocumentRecord
-from sonar.modules.institutions.api import InstitutionRecord
-
-
-def test_institution_resolver(client):
+def test_institution_resolver(document_fixture):
     """Test institution resolver."""
-    InstitutionRecord.create({
-        "pid": "usi",
-        "name": "Università della Svizzera italiana"
-    })
-
-    record = DocumentRecord.create({
-        "title": "The title of the record",
-        "institution": {"$ref": "https://sonar.ch/api/institutions/usi"}
-    })
+    assert document_fixture['institution'].get('$ref')
+    assert document_fixture['institution'][
+        '$ref'] == 'https://sonar.ch/api/institutions/org'
 
-    assert record.replace_refs().get('institution')['name'] == 'Università ' \
-        'della Svizzera italiana'
+    assert document_fixture.replace_refs().get(
```

Comment:
be careful with the resolvers. Changing the name can cause a huge number of document reindexing...

Reply:
OK, as discussed, I will do another PR to fine tune this resolvers and to prefer multiple call to backend for having the reference object information.